package.json

```diff
@@ -23,6 +23,14 @@
   ],
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
+  "exports": {
```

Comment:
I think we are missing `.js` extensions here, i.e.

```
"./logger": "./lib/logger.js",
"./logger/compat": "./lib/logger/compat.js",
"./lib/logger": "./lib/logger.js",
"./lib/logger/compat": "./lib/logger/compat.js"
```

Reply:
I'll add them, though they're clearly not necessary since tests pass. AFAICT it just shortens the lookup process since it doesn't guess  lib/logger.js before the (new) lib/logger/index.js.